### PR TITLE
Enhance ContractInterface instantiation

### PR DESCRIPTION
**Pull request by mialbu**
Allowing String parameter in ContractInterface ctor (big-endian script hash).

### CODE REVIEW

**Comment by gsmachado** [comment]
wow. 🤯

File: compiler/src/main/java/io/neow3j/compiler/converters/ObjectsConverter.java [hunk 1]
```diff
@@ -119,33 +118,49 @@ private AbstractInsnNode handleContractInterfaceCtor(AbstractInsnNode insn, NeoM
...
+                format("A constructor of a ContractInterface is required to take exactly one %s or %s type as " +
+                        "parameter.", getInternalName(Hash160.class), getInternalName(String.class)));
+    }
+
+    private void handleConstantScriptHash(NeoMethod neoMethod, NeoInstruction lastInstruction) {
+        if (lastInstruction.getOpcode().equals(OpCode.PUSHDATA1)) {
+            io.neow3j.types.Hash160 scriptHash = new io.neow3j.types.Hash160(new String(lastInstruction.getOperand()));
+            neoMethod.replaceLastInstruction(buildPushDataInsn(reverseArray(scriptHash.toArray())));
+        } else {
+            throw new CompilerException("Contract interface classes can only be initialized with Hash160 or " +
```
Comment (csmuller) [inline compiler/src/main/java/io/neow3j/compiler/converters/ObjectsConverter.java:141]: Please write an exception-test for this path.

**Review by csmuller — CHANGES_REQUESTED** [review]
Cool! Good stuff.
Left one comment.

File: compiler/src/main/java/io/neow3j/compiler/converters/ObjectsConverter.java (hunk 1 above)
Comment (mialbu) [reply compiler/src/main/java/io/neow3j/compiler/converters/ObjectsConverter.java]: I've added an UT in commit 5a366ff090cefc54e347e3b74732e2095acbff4e.

**Comment by mialbu** [comment]
Tests for `findstates` RPC fail, which will be fixed with PR #807. Otherwise, all tests pass.

**Pull request merged**